js/components/form/base-form.js

```diff
@@ -136,7 +136,7 @@ export default {
                 }
             },
             errorPlacement: function(error, element) {
-                $(element).closest('.form-group,.field-wrapper').append(error);
+                $(element).closest('.form-group,.field-wrapper').children('div').append(error);
```

Comment:
I confirm that it renders correctly:

![capture d ecran 2015-12-09 a 16 35 40](https://cloud.githubusercontent.com/assets/3556/11689590/f42feaf6-9e92-11e5-80b7-84de16a61b45.png)

![capture d ecran 2015-12-09 a 16 35 47](https://cloud.githubusercontent.com/assets/3556/11689593/f75ac20a-9e92-11e5-9803-72f7657e67e6.png)